### DISCUSSION TITLE
v0.4.10

### DISCUSSION THREAD

**Discussion by lbbrhzn** [lbbrhzn/ocpp#329] [Announcements]
## Changes

* Slider unavailable workaround (#328) @drc38
* tweak latency function (#327) @drc38

## What's Changed
* tweak latency function by @drc38 in https://github.com/lbbrhzn/ocpp/pull/327
* Slider unavailable workaround by @drc38 in https://github.com/lbbrhzn/ocpp/pull/328

**Full Changelog**: https://github.com/lbbrhzn/ocpp/compare/v0.4.9...v0.4.10

<hr /><em>This discussion was created from the release <a href='https://github.com/lbbrhzn/ocpp/releases/tag/v0.4.10'>v0.4.10</a>.</em>

**Comment by GrantKeymer** [comment]
Fantastic work guys!!!  The slider works and I can now set the charging current of my Pulsar Plus 🙌 

First I had to go into Developer Tools and set it to a value somewhere in the working range (I chose 16 Amps).  After that it stayed available and I can see the change from the slider reflected in the Wallbox app.  It takes only about 2 seconds for any change to appear.

Well I definitely owe you a coffee, beer or whatever it is now 🍻 👏 💯 🤸‍♂️

**Reply by drc38** [reply]
It's still somewhat of a mystery why it occurs for some HA installs and not others but good to hear the changes have sorted it for you.

**Reply by lbbrhzn** [reply]
I suspect it is somehow caused by RestoreEntity.
